x-pack/test_serverless/api_integration/test_suites/common/index_management/settings.ts

```diff
@@ -8,33 +8,32 @@
 import expect from '@kbn/expect';
 
 import { FtrProviderContext } from '../../../ftr_provider_context';
+import { RoleCredentials } from '../../../../shared/services';
 
 export default function ({ getService }: FtrProviderContext) {
-  const indexManagementService = getService('indexManagement');
-
-  describe('settings', () => {
-    let getIndexSettings: typeof indexManagementService['settings']['api']['getIndexSettings'];
-    let updateIndexSettings: typeof indexManagementService['settings']['api']['updateIndexSettings'];
-    let createIndex: typeof indexManagementService['indices']['helpers']['createIndex'];
-    let deleteAllIndices: typeof indexManagementService['indices']['helpers']['deleteAllIndices'];
+  const svlCommonApi = getService('svlCommonApi');
+  const svlUserManager = getService('svlUserManager');
+  const svlSettingsApi = getService('svlSettingsApi');
+  const svlIndicesHelpers = getService('svlIndicesHelpers');
+  let roleAuthc: RoleCredentials;
 
+  describe('settings', function () {
+    // see details: https://github.com/elastic/kibana/issues/187369
+    this.tags(['skipSvlOblt', 'skipSvlSearch', 'skipSvlSec']);
```

Comment:
Can you help me understand why you explicitly excluded the test from all project types? This is effectively the same as putting `describe.skip`, just more verbose (and IMO potentially confusing).

Reply:
https://github.com/elastic/kibana/pull/184105/commits/d252e088df7ecac873ebd12b1414b77e59c49177

Reply:
It's because it's failing in on pr ci and on mki, but not the entire suite.  The ones that fail, fail in all places and the the others pass in all places.

Reply:
Just to repeat and clarify: with these tags attached, none of the tests should run as you're skipping them for all project types. I don't know how you can see passing tests in there on CI - if that's really the case, it's a bug.

Reply:
https://github.com/elastic/kibana/pull/184105/commits/ae3caed5ea1578ec7e73e7b92a509b905d31e34a